src/components/ProductSummary.vue

```diff
@@ -38,28 +82,93 @@ const props = defineProps({
         type: Number,
         required: true,
     },
-    productVariantId: {
+    idOfShoppingCartItem: {
         type: String,
-        required: false,
+    },
+    count: {
+        type: Number,
+    },
+    maximumCount: {
+        type: Number,
+        default: 10,
     },
 })
 
+/**
+ * The emitted events of this component:
+ *
+ * @event count-changed - The count of the shopping cart item has changed.
+ * @event deleted - The shopping cart item has been deleted.
+ */
+const emit = defineEmits<{
+    (
+        event: 'count-changed',
```

Comment:
why not use a model for this? you can define a custom one using https://vuejs.org/api/sfc-script-setup.html#definemodel
then in the parent, you can just use v-model and do not have to handle the event differently (disclaimer: I did not yet look how you use this event)

Reply:
Same as above: How about I open a Gropius Issue for this so that I can change things later? Because I think there a more pressing issues right now. I have to say that I like both suggestions -- slot and custom model -- but I already put a lot of time into this issue here...

Reply:
this should be like 5 LOC to change
but if you want, fine for me